packages/cosmo/package.py

```diff
@@ -60,9 +66,10 @@ class Cosmo(MakefilePackage):
     patch('patches/5.07.mch1.0.p4/patch.Makefile', when='@5.07.mch1.0.p4')
     patch('patches/5.07.mch1.0.p4/patch.Makefile', when='@5.07.mch1.0.p5')
 
-    dycore_deps(apngit)
-    # Uncomment to build c2sm release 
-    # dycore_deps(c2smgit)
+    set_versions(apngit, reg_filter='.*mch.*')
+    set_versions(c2smgit)
+
+    set_dycore_dep()
```

Comment:
this can be removed, it is attaching the dycore version to the cosmo version, which you solved it already with the new spack command